datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -344,6 +362,10 @@ fn fetch_statistics(
                             table_idx,
                             stats,
                         )
+                    } else {
+                        // If none statistics of current column exists, set the Max/Min Accumulator to None.
```

Comment:
https://github.com/apache/arrow-datafusion/blob/807b7a5f7eb858e9f7162e1f00ffeeedd0bf2050/datafusion/core/src/datasource/file_format/parquet.rs#L312
Here both `max_values` and `min_values ` are accumulator for all row_groups

Set  `max_values[i] = None` in loop `for row_group_meta in meta_data.row_groups() {}` assume that  we will no longer need this col static. So we need a short cut (No need read next row_group metadata for this col avoid set another value)

Reply:
If i miss something, plz tell me

Reply:
> If i miss something, plz tell me

Once max_values and min_values are set to none, even if the column statistics exists in next row_group, summarize_min_max will do nothing because the max_values and min_values are none.

Reply:
If there are no statistics in one of the row groups, I wonder if it is correct to keep going 🤔

Reply:
> If there are no statistics in one of the row groups, I wonder if it is correct to keep going 🤔

Are there any statistics for other columns in current row groups?

Reply:
> > If i miss something, plz tell me
> 
> Once max_values and min_values are set to none, even if the column statistics exists in next row_group, summarize_min_max will do nothing because the max_values and min_values are none.
Ohh, thanks i will check later.